documentation/reference/qfieldcloud/concepts.md

```diff
@@ -8,19 +8,19 @@ To interact with QFieldCloud you need a registered user. Each user can create, m
 
 ## Projects
 
-Projects are the main data container within QFieldCloud. Each user can create one or more QFieldCloud projects. Each project contains a single `.qgs`/`.qgz` QGIS project file, the geospatial files - geopackages, shapefiles, tiffs, and additional data such as photos, PDFs etc. Note all the project data should be within a single QFieldCloud project.
+Projects are the main data container within QFieldCloud. Each user can create one or more QFieldCloud projects. Each project contains a single `.qgs`/`.qgz` QGIS project file, the geospatial files - GeoPackages, Shapefiles, TIFs, and additional data such as photos, PDFs etc. All the project data must be within a single QFieldCloud project.
 
 ### Project collaborators
 
-A project collaborator is a user invited to contribute to a project. One project may have multiple collaborators. Collaborators with role **owner** or **admin** can add more users as collaborators. If the project is owned by an organization, you can also add **teams** as collaborators.
+A project collaborator is another QFieldCloud user invited to contribute to a project. One project may have multiple collaborators. Collaborators with role **owner** or **admin** can add more users as collaborators. If the project is owned by an organization, you can also add **teams** as collaborators. Read more about collaborator roles [here](permissions.md).
```

Comment:
Can we make the "Read more aboutcollaborator roles" text itself the link and remove "here"?

Reply:
I am not super fan of sentence links, but I see your point to avoid "here" too.

Reply:
Failed to rephrase it in a sensible manner, so I followed your advice.